consensus/dummy/consensus_test.go

```diff
@@ -164,30 +155,19 @@ func TestVerifyBlockFee(t *testing.T) {
 			extraStateContribution: nil,
 			shouldErr:              false,
 		},
-		"parent time > current time": {
-			baseFee:                big.NewInt(100),
-			parentBlockGasCost:     big.NewInt(0),
-			parentTime:             11,
-			currentTime:            10,
-			txs:                    nil,
-			receipts:               nil,
-			extraStateContribution: big.NewInt(10_000_000),
-			shouldErr:              false,
-		},
 	}
 
 	for name, test := range tests {
 		t.Run(name, func(t *testing.T) {
-			blockGasCost := calcBlockGasCost(
-				ApricotPhase4TargetBlockRate,
-				ApricotPhase4MinBlockGasCost,
-				ApricotPhase4MaxBlockGasCost,
-				ApricotPhase4BlockGasCostStep,
+			blockGasCost := header.BlockGasCostWithStep(
 				test.parentBlockGasCost,
-				test.parentTime, test.currentTime,
+				ap4.BlockGasCostStep,
+				test.timeElapsed,
 			)
+			bigBlockGasCost := new(big.Int).SetUint64(blockGasCost)
```

Comment:
tbh this test should probably just specify the `blockGasCost` rather than calculating it - but it didn't feel relevant to this PR.